assembly/__tests__/algebraic_operations.spec.ts

```diff
@@ -48,12 +48,26 @@ describe("Algebraic operations", () => {
     for (let i = 0; i < testCases.length; i++) {
       const testCase = testCases[i];
       const x = BigInt.fromString(testCase.x);
-      const actual = x.pow(<i32>3);
+      let actual = x.pow(<i32>3);
+      actual = x.pow(<i64>3);
+      actual = x.pow(<i16>3);
```

Comment:
Not sure we want to overwrite previous results before checking them for correctness here.

Reply:
I've changed the tests to test the values for different integer types as well.